### PR TITLE
Update install on GKE doc with settings and guidelines used during ben…

### PR DESCRIPTION
**Pull request by johnsmartco**
…chmarks

**What this PR does**:

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [ ] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [ ] CHANGELOG.md updated (not required for documentation PRs)
- [ ] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: docs/content/en/install/gke/_index.md [hunk 1]
```diff
@@ -43,16 +43,60 @@ On this infrastructure the K8ssandra installation will consist of the following
...
+
+While the section above includes infrastructure settings for **minimum** production workloads, performance benchmarks reveal a wider range of recommendations that are important to consider. The performance benchmark report, available in this [blog](https://k8ssandra.io/blog/articles/k8ssandra-performance-benchmarks-on-cloud-managed-kubernetes/), compared throughput and latency of:
+
+* The baseline performance of a Cassandra cluster running on AWS EC2 instances -- a common setup for enterprises operating Cassandra clusters
+* With the performance of K8ssandra running on AWS, EKS, and GCP GKE
+
+It's important to note the following additional infrastructure recommendations from the benchmark:
+
+* 8 to 16 vCPUs 
+  * r5 instances: Intel Xeon Platinum 8000 series
```
Comment (adejanovski) [inline docs/content/en/install/gke/_index.md:63]: Should we mention AWS instance types in a GKE doc?

File: docs/content/en/install/gke/_index.md [hunk 2]
```diff
@@ -43,16 +43,60 @@ On this infrastructure the K8ssandra installation will consist of the following
...
+* 5k to 10k Input/Output Operations per Second (IOPS)
+
+For the disk performance, the benchmark used [Cassandra inspired fio profiles](https://github.com/ibspoof/cassandra-fio) that attempt to emulate Leveled Compaction Strategy and Size Tiered Compaction Strategy behaviors. In the report, throughput and latency results were measured for three types of GCP disks:
+
+* GCP GKE n2-highmem-8, pd-ssd disk (`premium-rwo`)
+* GCP GKE n2-highmem-8, pd-balanced disk (`standard-rwo`)
+* GCP GKE e2-highmem-8, pd-ssd disk (`premium-rwo`)
+
+{{% alert title="Disk recommendations" color="success" %}}
+The e2 instance class in GCP is the default in GKE, but proved to be under-powered compared to the AWS r5 instance class with approximately 20% lower [sysbench](https://github.com/akopytov/sysbench) score. The n2 instance class offered much better performance with a score that was approximately 10% higher than the AWS r5 instances. The pd-balanced (standard-rwo) disks provided enough performance for this specific benchmarks; thus we didn’t need to upgrade to the slightly more expensive pd-ssd (premium-rwo) disks.
```
Comment (adejanovski) [inline docs/content/en/install/gke/_index.md:76]: Same question here. Maybe we could phrase things to a more generic "other cloud vendors last generation instances"?

File: docs/content/en/install/gke/_index.md [hunk 3]
```diff
@@ -43,16 +43,60 @@ On this infrastructure the K8ssandra installation will consist of the following
...
+
+For more, see this [troubleshooting tip]({{< relref "tasks/troubleshoot/#check-quotas" >}}).
+{{% /alert %}}
+
+## Infrastructure and Cassandra recommendations
+
+While the section above includes infrastructure settings for **minimum** production workloads, performance benchmarks reveal a wider range of recommendations that are important to consider. The performance benchmark report, available in this [blog](https://k8ssandra.io/blog/articles/k8ssandra-performance-benchmarks-on-cloud-managed-kubernetes/), compared throughput and latency of:
+
+* The baseline performance of a Cassandra cluster running on AWS EC2 instances -- a common setup for enterprises operating Cassandra clusters
+* With the performance of K8ssandra running on AWS, EKS, and GCP GKE
```
Comment (adejanovski) [inline docs/content/en/install/gke/_index.md:58]:
```suggestion
* With the performance of K8ssandra running on AWS EKS, and GCP GKE
```

File: docs/content/en/install/gke/_index.md (hunk 1 above)
Comment (johnsmartco) [reply docs/content/en/install/gke/_index.md:63]: Oh, right. Thanks @adejanovski. Which instance type should I list for a GKE cluster in production that we'd suggest users consider?

File: docs/content/en/install/gke/_index.md (hunk 2 above)
Comment (johnsmartco) [reply docs/content/en/install/gke/_index.md:76]: Good idea - I'll change this to "...proved to be under-powered, compared to other cloud vendors latest generation instances, with approximately 20% lower..."

and:   "The n2 instance ... 10% higher than other cloud vendors instances."

Comment (johnsmartco) [reply docs/content/en/install/gke/_index.md]: See updated text, thanks.

File: docs/content/en/install/gke/_index.md (hunk 1 above)
Comment (johnsmartco) [reply docs/content/en/install/gke/_index.md]: See updated text, thanks.

**Review by adejanovski — APPROVED** [review]

**Pull request merged**